scripts/install/SetupStorage.php

```diff
@@ -39,6 +39,7 @@ class SetupStorage extends InstallAction
      */
     public function __invoke($params)
     {
+
         $fsService = $this->getServiceLocator()->get(FileSystemService::SERVICE_ID);
         if (!$fsService->hasDirectory(StorageService::FILE_SYSTEM_ID)) {
```

Comment:
We don't need this empty line. If you want to add an empty line it will be better to add it before the `if` statement.
```suggestion
    {
        $fsService = $this->getServiceLocator()->get(FileSystemService::SERVICE_ID);

        if (!$fsService->hasDirectory(StorageService::FILE_SYSTEM_ID)) {
```